### PR TITLE
re-nerfs the venus

### PR DESCRIPTION
**Pull request by jjpark-kb**
## About The Pull Request
vines now heal less when they hit a non-dead living thing.
vines now heal when ON vines and take damage when OFF vines.
vines now do not heal when walking through vines.

## How This Contributes To The Skyrat Roleplay Experience
vines are a bit strong-- addressing venus will make vines more tolerable.

## Changelog

:cl:
balance: vines heal less when hurting alive things
add: vines now take damage when off vines / heal when on vines
del: vines no longer heal while moving through vines
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm [hunk 1]
```diff
@@ -245,6 +246,13 @@
 		to_chat(src, span_boldwarning("You cannot drag living things!"))
 		return
 	return ..()
+
+/mob/living/simple_animal/hostile/venus_human_trap/proc/check_vines()
```
Comment (Zonespace27) [inline code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm]: modularize this

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
I agree with Zonespace, otherwise this is good to go!

**Comment by YakumoChen** [comment]
Fairly extreme nerf unless you make on-vine healing much higher or give them a large health buff. 

The health regen is the only factor that keeps Venus competitive, they are very fragile and can only beat back the crew through attrition. This will make them die extremely quickly

File: modular_skyrat/modules/apocolypse_of_scythes/code/venus.dm [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+/mob/living/simple_animal/hostile/venus_human_trap/death(gibbed)
+	for(var/i in vines)
+		qdel(i)
+	return ..()
+
+/mob/living/simple_animal/hostile/venus_human_trap/start_pulling(atom/movable/AM, state, force, supress_message)
+	if(isliving(AM))
+		to_chat(src, span_boldwarning("You cannot drag living things!"))
+		return
+	return ..()
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/apocolypse_of_scythes/code/venus.dm]: Single (or double here) letter var bad.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Review by GoldenAlpharex — APPROVED** [review]

**Pull request merged**